api-publishing/src/main/java/org/zalando/nakadi/EventPublishingController.java

```diff
@@ -115,6 +128,11 @@ private ResponseEntity postBinaryEvents(final String eventTypeName,
             throw new BlockedException("Application or event type is blocked");
         }
 
+        final EventType eventType = eventTypeCache.getEventType(eventTypeName);
+
+        // throws exception, maybe move it somewhere else
+        authValidator.authorizeEventTypeWrite(eventType);
```

Comment:
What happens if it really throws an exception?  Is it handled properly?

Reply:
it is handled by handler https://github.com/zalando/nakadi/blob/master/core-services/src/main/java/org/zalando/nakadi/controller/advice/NakadiProblemExceptionHandler.java#L94

Reply:
So let's remove the comment if we don't need to move it?